### PR TITLE
partial for direct download links on episode page

### LINKED ISSUE #138: Show page - Direct Downloads - links are to current page, not mp3/Youtube, etc
image

The links to various "Direct Downloads" point to the current page/episode when they should point to their respective downloads.

Current code being rendered:
`Direct Download: [MP3](https://jupiterbroadcasting.net/show/linux-unplugged/465/) | [Video](https://jupiterbroadcasting.net/show/linux-unplugged/465/) | [YouTube](https://jupiterbroadcasting.net/show/linux-unplugged/465/)`

### PR DESCRIPTION
**Pull request by aaronrea**
fixes #138

### CODE REVIEW

**Comment by gerbrent** [comment]
nice one @aaronrea !

**Review by StefanS-O — APPROVED** [review]

File: themes/jb/layouts/partials/episode/downloadlinks.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+| <a href="{{.Params.podcast_ogg_file}}">OGG</a>
+{{ end }}
+{{ if (not (eq .Params.video_file nil)) }}
+| <a href="{{.Params.video_file}}">Video</a>
+{{ end }}
+{{ if (not (eq .Params.video_hd_file nil)) }}
+| <a href="{{.Params.video_hd_file}}">HD Video</a>
+{{ end }}
+{{ if (not (eq .Params.video_mobile_file nil)) }}
+| <a href="{{.Params.vide_mobile_file}}">Mobile Video</a>
```
Comment (elreydetoda) [inline themes/jb/layouts/partials/episode/downloadlinks.html:15]: Looks like there was a slight spelling error here: the line before this you had `.Params.video_mobile_file` and this line you put `vide` instead.

@aaronrea would you mind doing another small PR to correct?

**Review by elreydetoda — CHANGES_REQUESTED** [review]
Beside the one comment, very nicely done 😁 I love this addition 🚀

File: themes/jb/layouts/partials/episode/downloadlinks.html (hunk 1 above)
Comment (StefanS-O) [reply themes/jb/layouts/partials/episode/downloadlinks.html:15]: @elreydetoda good catch!

Comment (aaronrea) [reply themes/jb/layouts/partials/episode/downloadlinks.html:15]: good catch and done. #188  thanks!

**Pull request merged**